talks/broad_12_mar_2018.md

```diff
@@ -0,0 +1,558 @@
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.2.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.3.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.4.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.5.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.6.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.7.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.8.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.9.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.10.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.11.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.12.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.13.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.14.png){ width="100%" }
+
+-------------------
+
+![Simplify example](simplify_walkthrough/simplify_walkthrough.15.png){ width="100%" }
+
+
+## Wright-Fisher, N=10: before simplification
+
+![Wright-Fisher tree sequence](sim_wf.anim.gif)
+
+## Wright-Fisher, N=10: before simplification
+
+![Wright-Fisher tree sequence](sim_wf_unlabeled.anim.gif)
+
+## ... and after simplification
+
+![Simplified Wright-Fisher tree sequence](sim_wf_simplified.anim.gif)
+
+
+## *Revised* tree recording strategy
+
+Every time an individual is born, we must:
+
+
+1. add each gamete to the Node Table,
+2. add entries to the Edge Table
+    recording which parent each gamete inherited each bit of genome from
+3. add any new mutations to the Mutation Table 
+    and (if necessary) their locations to the Site Table.
+
+... and,
+
+4. Every so often, *simplify* the tables so far,
+    retaining the history of the current generation.
+
+
+<!-- 5. display impressive speedups 2min) -->
+# Implementation and results
+
+## Benchmark implementation
+
+- Recording, simplifying, and output of tables: 
+    `C` code in `msprime`. (soon: `tskit`)
+
+- Simulation: [`fwdpp`](https://github.com/molpopgen/fwdpp), by Kevin Thornton (in `C++`) ([code](https://github.com/molpopgen/fwdpy11_arg_example))
+
+- Glue: [`pybind11`](https://github.com/pybind/pybind11/) and [`numpy`](http://www.numpy.org/)
+
+- Machine: Ubuntu / 2x 2.6 GHz Intel E5-2650 CPU
+
+. . .
+
+*Other implementations:* 
+
+- pure `python`, interfacing with [`simuPOP`](https://github.com/BoPeng/simuPOP)
```

Comment:
might as well link to https://github.com/ashander/ftprime here

Reply:
ah yep